emily/handler/tests/integration/util/constants.rs

```diff
@@ -2,13 +2,11 @@
 
 use emily_handler::api::models::{common::Status, withdrawal::WithdrawalParameters};
 
-// TODO(273):  Remove the "local" prefix once we figure out why all local
-// testing calls seem to forcibly start with `local`.
-pub const EMILY_ENDPOINT: &'static str = "http://localhost:3031/local";
-pub const EMILY_WITHDRAWAL_ENDPOINT: &'static str = "http://localhost:3031/local/withdrawal";
-pub const EMILY_DEPOSIT_ENDPOINT: &'static str = "http://localhost:3031/local/deposit";
-pub const EMILY_CHAINSTATE_ENDPOINT: &'static str = "http://localhost:3031/local/chainstate";
-pub const EMILY_TESTING_ENDPOINT: &'static str = "http://localhost:3031/local/testing";
+pub const EMILY_ENDPOINT: &'static str = "http://localhost:3031";
+pub const EMILY_WITHDRAWAL_ENDPOINT: &'static str = "http://localhost:3031/withdrawal";
+pub const EMILY_DEPOSIT_ENDPOINT: &'static str = "http://localhost:3031/deposit";
+pub const EMILY_CHAINSTATE_ENDPOINT: &'static str = "http://localhost:3031/chainstate";
+pub const EMILY_TESTING_ENDPOINT: &'static str = "http://localhost:3031/testing";
```

Comment:
I think I understand. We need the local prefix for local lambda with SAM, but we can don't do integration tests with SAM, just regular warp, so can ditch the local prefix. Is that right?

Reply:
Yes. Sam forces us to have it.